10-algorithms.Rmd

```diff
@@ -79,6 +80,7 @@ By default RStudio 'code-checks' R scripts and underlines faulty code with a red
 knitr::include_graphics("figures/codecheck.png")
 ```
 
+<!-- JM: the reprex package is great especially for communicating on Github, via twitter, etc. but can you really use it to test if your own code is reproducible?-->
```

Comment:
Yes, but agree it's most useful for communicating to others how to get to a given state and to ask questions or illustrated bugs. That may be worth mentioning. How about changing the first line of the note to read:

```
 A useful tool for reproducibility and communication is the **reprex** package. 
```

Reply:
Ok